### DISCUSSION TITLE
Deploying to Heroku

### DISCUSSION THREAD

**Discussion by mig-25** [parcel-bundler/parcel#4835] [General] [answered]
Hi,

I made a build of my application to upload it to Heroku.
When I made a `git push heroku master`
I was able to make a build, but I am getting a Application error when loading the page.

I added a package.json file in the build folder so Heroku could treat it as node.js app. But I am not quite sure what sorts of scripts should be used.
In my dev folder, in my package.json I am making use of pug and sass. But I am uploading the build folder to Heroku.

This is the content of my package.json in my build folder:
```
{
  "name": "damisis-test-deploy",
  "version": "1.0.0",
  "description": "",
  "scripts": {
    "start": "node index.js"
  },
    "engines": {
    "node": "12.x"
  },
  "keywords": [],
  "author": "",
  "license": "ISC"
}
```
This is log from the build:

> 2020-07-02T16:11:38.082730+00:00 heroku[web.1]: State changed from crashed to starting
> 2020-07-02T16:11:41.356589+00:00 heroku[web.1]: Starting process with command `npm start`
> 2020-07-02T16:11:44.416993+00:00 heroku[web.1]: Process exited with status 0
> 2020-07-02T16:11:44.462560+00:00 heroku[web.1]: State changed from starting to crashed
> 2020-07-02T16:11:44.255116+00:00 app[web.1]: 
> 2020-07-02T16:11:44.255137+00:00 app[web.1]: > damisis-test-deploy@1.0.0 start /app
> 2020-07-02T16:11:44.255138+00:00 app[web.1]: > node index.js
> 2020-07-02T16:11:44.255138+00:00 app[web.1]: 
> 2020-07-02T16:13:55.958041+00:00 heroku[router]: at=error code=H10 desc="App crashed" method=GET path="/" host=damisis-test.herokuapp.com request_id=3ff51e79-2b54-4cdb-8092-605b7a2d4ca0 fwd="188.151.64.163" dyno= connect= service= status=503 bytes= protocol=https

thanks

**Comment by mischnic** [comment, marked as answer]
This isn't related to Parcel.

You're trying to deploy a static website to Heroku. Searching for "heroku static website" on Google, it looks like there no real answer...

**Comment by mig-25** [comment]
Sorry, my bad. But I took your suggestion and found a solution :)

Thanks

On 2 Jul 2020, at 19:33, Niklas Mischkulnig <notifications@github.com> wrote:

﻿

This isn't related to Parcel.

You're trying to deploy a static website to Heroku. Searching for "heroku static website" on Google, it looks like there no real answer...
